### PR TITLE
Sample that uses EXT_lights_image_based extension

### PR DESCRIPTION
**Pull request by MiiBond**

### CODE REVIEW

File: 2.0/EnvironmentTest/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Metal-Rough Spheres with IBL
```
Comment (emackey) [inline 2.0/EnvironmentTest/README.md]: This got renamed.

File: 2.0/EnvironmentTest/glTF-EXT_lights_image_based/EnvironmentTest.gltf [hunk 2]
```diff
@@ -0,0 +1,742 @@
...
+                "aspectRatio": 1.3333333730697632
+            },
+            "type": "perspective",
+            "name": "render_camera"
+        }
+    ],
+    "images": [
+        {
+            "name": "tmp_image_pie_dc1e_1a22_fbf9roughness_map_roughness_tmp_image_pie_dc1e_1a22_fbf9metal_map_metallic_0",
+            "uri": "EnvironmentTest_images/tmp_image_pie_dc1e_1a22_fbf9roughness_map_roughness_tmp_image_pie_dc1e_1a22_fbf9metal_map_metallic_0.jpg",
```
Comment (emackey) [inline 2.0/EnvironmentTest/glTF-EXT_lights_image_based/EnvironmentTest.gltf]: These images were renamed as well.

**Comment by emackey** [comment]
Can we also rename the folder `glTF-EXT_lights_image_based` to just `glTF-IBL`?  Thanks.

**Comment by cx20** [comment]
I added this model to [gltf-test](https://github.com/cx20/gltf-test#extension-test-models). I think the current status is as follows.
image

**Pull request merged**